certified-connectors/The Bot Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,329 @@
+		"version": "1.0",
+		"contact": {
+			"name": "The Bot Platform Support",
+			"url": "https://thebotplatform.com/https-thebotplatform-com-contact/",
+			"email": "hello@thebotplatform.com"
+		}
+	},
+	"host": "api.thebotplatform.com",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"x-ms-connector-metadata": [{
+			"propertyName": "Website",
+			"propertyValue": "https://thebotplatform.com"
+		},
+		{
+			"propertyName": "Privacy policy",
+			"propertyValue": "https://thebotplatform.com/privacy-policy/"
+		},
+		{
+			"propertyName": "Categories",
+			"propertyValue": "Communication;Website"
+		}
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/v1.0/activity/external": {
+			"post": {
+				"responses": {
+					"201": {
+						"description": "created",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"$ref": "#/definitions/ExternalActivity"
+						}
+						}
+						}
+					}
+					},
+				"summary": "Send a simple text message to a bot user",
+				"operationId": "SimpleTextMessage",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"data": {
+								"type": "object",
+								"properties": {
+									"attributes": {
+										"type": "object",
+										"properties": {
+											"messages": {
+												"type": "array",
+												"items": {
+													"type": "object",
+													"properties": {
+														"text": {
+															"type": "string",
+															"description": "Enter your text message here",
+															"title": "",
+															"format": ""
+														}
+													}
+												},
+												"description": "messages"
+											},
+											"recipient": {
+												"type": "string",
+												"description": "email address of recipient",
+												"title": "email address"
+											}
+										},
+										"description": "attributes"
+									},
+									"type": {
+										"type": "string",
+										"description": "type",
+										"title": "",
+										"default": "external-activity",
+										"enum": [
+											"external-activity"
+										],
+										"x-ms-visibility": "internal"
+									}
+								},
+								"description": "data",
+								"required": [
+									"type"
+								]
+							}
+						},
+						"required": [
+							"data"
+						]
+					}
+				}],
+				"description": "Send a message to a bot user using text"
+			}
+		},
+		"/v1.0/userattributes": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"type" : "array",
+									"items": {
+										"$ref": "#definitions/UserAttribute"
```

Comment:
@tbpsami The schema reference '#definitions/UserAttribute' is not valid. Schema references must start with '#/definitions/' and be valid JSON Pointers. Please fix and define the reference in the definition section.

Reply:
Defined schema 'UserAttribute' in definitions section